packages/manager/apps/web-domains/src/alldoms/components/ServiceDetail/ServiceDetailSubscribing/ServiceDetailSubscribing.component.tsx

```diff
@@ -37,19 +47,17 @@ export default function ServiceDetailSubscribing({
 
       <div className="flex items-center justify-between">
         <ServiceDetailSubscribingRenewDate
-          expirationDate={serviceInfoDetail.serviceInfo.billing.expirationDate}
-          renewMode={serviceInfoDetail.serviceInfo.billing.renew.current.mode}
-          renewDate={
-            serviceInfoDetail.serviceInfo.billing.renew.current.nextDate
-          }
-          serviceState={serviceInfoDetail.allDomResourceState}
+          expirationDate={expirationDate}
+          renewMode={renewMode}
+          renewDate={renewalDate}
+          serviceState={allDomResourceState}
         />
 
         <ServiceActionMenu
-          serviceId={`${serviceId}-subscribe`}
-          serviceName={serviceInfoDetail.allDomResource.currentState.name}
+          id={`${currentState?.name}-${ActionEnum.OnlyRenew}`}
```

Comment:
```suggestion
          id={`${currentState.name}-${ActionEnum.OnlyRenew}`}
```
`currentState` seems to be always defined